README.md

```diff
@@ -554,6 +554,8 @@ A curated list of awesome Java frameworks, libraries and software.
 * [consul-api](https://github.com/Ecwid/consul-api) - Client for the [Consul](https://www.consul.io/) API a distributed, highly available and datacenter-aware registry/discovery service.
 * [Eureka](https://github.com/Netflix/eureka) - REST based service registry for resilient load balancing and failover.
 * [Lagom](https://www.lightbend.com/lagom) - Framework for creating microservice-based systems.
+* [Light Java](https://github.com/networknt/light-java) - A fast and configurable REST microservices framework for docker containers on the cloud.
```

Comment:
The main point is "docker containers on the cloud"? I could run struts in a container in the cloud and it wouldn't be better.